api-reference/v1.0/api/domain_forcedelete.md

```diff
@@ -0,0 +1,86 @@
+# domain: forceDelete
```

Comment:
Please shelve this topic.  Unfortunately this API did not make it to v1.0 so please remove.  We'll add it back ASAP, but it'll take a deployment cycle.